cartography/data/jobs/cleanup/aws_import_codeguru_associations_cleanup.json

```diff
@@ -0,0 +1,11 @@
+{
+    "statements": [
+      {
+        "query": "MATCH (:AWSAccount{id: {AWS_ID}})-[:RESOURCE]->(cga:CodeguruAssociation) WHERE cga.lastupdated <> {UPDATE_TAG} WITH cga LIMIT {LIMIT_SIZE} DETACH DELETE (cga) return COUNT(*) as TotalCompleted",
+        "iterative": true,
+        "iterationsize": 100
+      }
+    ],
+    "name": "cleanup Codeguru"
+  }
+  
```

Comment:
@t-maoshengye It looks like you need to setup pre-commit. This file is missing EOF, and that would have been fixed by pre-commit